api/components/text.schema.json

```
@@ -17,6 +17,32 @@
         },
         "style": {
             "$ref": "defs/textStyle.schema.json"
+        },
+        "locale": {
+            "description": "the value of the language for the text",
```

```suggestion
            "description": "The value of the language for the text.",
```
We should be careful when writing descriptions for properties/components as we will generate a documentation from that later. It would be better if each description was nicely styled.
Can you please apply these changes across the whole PR ?